ext/mbstring/mbstring.c

```diff
@@ -4733,46 +4807,17 @@ MBSTRING_API char *php_mb_safe_strrchr(const char *s, unsigned int c, size_t nby
 	return last;
 }
 
-MBSTRING_API size_t php_mb_stripos(int mode, const char *old_haystack, size_t old_haystack_len, const char *old_needle, size_t old_needle_len, zend_long offset, const mbfl_encoding *enc)
+MBSTRING_API size_t php_mb_stripos(int mode, zend_string *haystack, zend_string *needle, zend_long offset, const mbfl_encoding *enc)
```

Comment:
`mode` is declared as int, but should it not be bool?

Reply:
You are right. Thanks for the catch.

Fixed.